Build problem when not on the DAS5 fs5 headnode: make[2]: *** No rule to make target `/usr/lib64/libopenblas.so', needed by `dist/bin/sagecal'

**Describe the bug**
Cannot build SAGECal when libopenblas.so is not in /usr/lib64/
> make[2]: *** No rule to make target `/usr/lib64/libopenblas.so', needed by `dist/bin/sagecal'.  Stop.

**To Reproduce**
Steps to reproduce the behavior:
1. ssh to fs5 headnode on DAS5
2. ssh to node
3. cd sagecal-code/build
4. `module load casacore/2.3.0-gcc-4.9.3`
5. `module load cmake/3.8.2`
6. `cmake .. -DCMAKE_INSTALL_PREFIX=/home/hspreeuw/sagecal-code/src/MS -DHAVE_CUDA=FALSE`
7. `make` ---> ```No rule to make target `/usr/lib64/libopenblas.so'``` 
8. `ls /usr/lib64/libopenblas.so`----> ```ls: cannot access /usr/lib64/libopenblas.so: No such file or directory```. On the headnode, we do have a file ```/usr/lib64/libopenblas.so```
9. `module load openblas`
10.  `cmake .. -DCMAKE_INSTALL_PREFIX=/home/hspreeuw/sagecal-code/src/MS -DHAVE_CUDA=FALSE`
11. `make`---> ```No rule to make target `/usr/lib64/libopenblas.so'```

**Expected behavior**
Successful build

**Additional context**
This is the gpu_support branch. Build for CPU.

Issue still seems to be present on the dev branch, when not on headnode, after `make`:

> Scanning dependencies of target sagecal_gpu
[ 83%] Building CXX object src/MS/CMakeFiles/sagecal_gpu.dir/data.cpp.o
[ 84%] Building CXX object src/MS/CMakeFiles/sagecal_gpu.dir/main.cpp.o
[ 85%] Linking CXX executable ../../dist/bin/sagecal_gpu
/usr/bin/ld: cannot find -lopenblas
collect2: error: ld returned 1 exit status
make[2]: *** [dist/bin/sagecal_gpu] Error 1
make[1]: *** [src/MS/CMakeFiles/sagecal_gpu.dir/all] Error 2
make: *** [all] Error 2

@SarodYatawatta can you confirm?

--- a/src/MS/CMakeLists.txt
+++ b/src/MS/CMakeLists.txt
@@ -42,7 +42,6 @@ if(HAVE_CUDA)
         ${WCSLIB_LIBRARIES}
         ${GLIB_PKG_LIBRARIES}
         ${LIBGFORTRAN_LIBRARIES}
-        -lopenblas
         -lgfortran
         -lpthread
         -lm

